vpr/src/base/read_options.cpp

```diff
@@ -1677,6 +1677,12 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("0")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.enable_analytic_placer, "--enable_analytic_placer")
+        .help(
+            "Enables analytic placer, and skips simulated annealing. The result in improved by quenching")
```

Comment:
"result is improved by quenching"

Reply:
Probably should rewrite this to say "Enables the analytic placer. Once analytic placement is done, the result passed through the quench phase of the annealing placer for local improvement".